src/mixins/resource-lookup-mixin.ts

```diff
@@ -8,10 +8,10 @@ import { FilingCodes } from '@/enums'
  */
 @Component({
   computed: {
-    ...mapState(['configObject', 'isBComp'])
```

Comment:
Bug! `isBComp` is not a state. I don't know how this worked before because it was always falsy ...?